### PR TITLE
[CHI-373] Added picker group text+icon labels

### PR DESCRIPTION
**Pull request by janibolkvadze**
@mattnickles @jllr 

https://ctl.atlassian.net/browse/CHI-373

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/260/ for more details.

File: src/website/views/components/picker-group.pug [hunk 1]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
       </label>
     </div>
 
+h3 Texts and icons
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please change to h4

File: src/website/views/components/picker-group.pug [hunk 2]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom"&gt;&lt;/i&gt;</code>.
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please remove this description as I think it is not necessary. @mattnickles thoughts?

File: src/website/views/components/picker-group.pug [hunk 3]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup
+      input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+      label(for=`ib-radio1`)
+        i.a-icon.icon-atom
+        span
+          = `Radio1`
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze can you remove this line and move to line 42 with this format span Radio 1

File: src/website/views/components/picker-group.pug [hunk 4]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+      input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+      label(for=`ib-radio2`)
+        span
+          = `Radio2`
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze can you remove this line and move to line 46 with this format span Radio 2

File: src/website/views/components/picker-group.pug [hunk 5]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+        i.a-icon.icon-atom
+      input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+      label(for=`ib-radio3`)
+        i.a-icon.icon-atom
+        span
+          = `Radio3`
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze can you remove this line and move to line 52 with this format span Radio 3

File: src/website/views/components/picker-group.pug [hunk 6]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+      <input class="a-picker" type="radio" name="radios" id="radio3">
+      <label for="radio3">
+        <i class="a-icon icon-atom"></i>
+        <span>Radio3</span>
+        <i class="a-icon icon-atom"></i>
+      </label>
+    </div>
+
+
+h3 Icons
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please change to h4

File: src/website/views/components/picker-group.pug [hunk 7]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom">&lt;/i></code>.
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please remove this description as I think it is not necessary. @mattnickles thoughts?

File: src/website/views/components/picker-group.pug [hunk 8]
```diff
@@ -150,6 +230,20 @@ p.-text
...
       </label>
     </div>
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio-i-1">
+      <label class="-sm" for="radio-i-1">
+        <i class="a-icon icon-atom -xs"></i>
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-2">
+      <label class="-sm" for="radio-i-2">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -xs class

File: src/website/views/components/picker-group.pug [hunk 9]
```diff
@@ -150,6 +230,20 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-3">
+      <label class="-sm" for="radio-i-3">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -xs class

File: src/website/views/components/picker-group.pug [hunk 10]
```diff
@@ -165,6 +259,20 @@ p.-text
         Radio3
       </label>
     </div>
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio-i-1">
+      <label class="-md" for="radio-i-1">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 11]
```diff
@@ -165,6 +259,20 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-2">
+      <label class="-md" for="radio-i-2">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 12]
```diff
@@ -165,6 +259,20 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-3">
+      <label class="-md" for="radio-i-3">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 13]
```diff
@@ -179,6 +287,19 @@ p.-text
       <label class="-lg" for="radio3">
         Radio3
       </label>
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio-i-1">
+      <label class="-lg" for="radio-i-1">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 14]
```diff
@@ -179,6 +287,19 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-2">
+      <label class="-lg" for="radio-i-2">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 15]
```diff
@@ -179,6 +287,19 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-3">
+      <label class="-lg" for="radio-i-3">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 16]
```diff
@@ -194,4 +315,17 @@ p.-text
       <label class="-xl" for="radio3">
         Radio3
       </label>
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio-i-1">
+      <label class="-xl" for="radio-i-1">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 17]
```diff
@@ -194,4 +315,17 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-2">
+      <label class="-xl" for="radio-i-2">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 18]
```diff
@@ -194,4 +315,17 @@ p.-text
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio-i-3">
+      <label class="-xl" for="radio-i-3">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: test/chi/components/picker-group.pug [hunk 19]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+      label(for=`ia-${type}`)
         = type
+.test-with-texts-and-icons
+  h3 Texts and icons
+  .m-pickerGroup
+    input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+    label(for=`ib-radio1`)
+      i.a-icon.icon-atom
+      span
+        = `Radio1`
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze can you remove this line and move to line 27 with this format span Radio 1

File: test/chi/components/picker-group.pug [hunk 20]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+    input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+    label(for=`ib-radio2`)
+      span
+        = `Radio2`
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze can you remove this line and move to line 31 with this format span Radio 2

File: test/chi/components/picker-group.pug [hunk 21]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+      i.a-icon.icon-atom
+    input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+    label(for=`ib-radio3`)
+      i.a-icon.icon-atom
+      span
+        = `Radio3`
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze can you remove this line and move to line 37 with this format span Radio 3

File: test/chi/components/picker-group.pug [hunk 22]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
-      label(for=`i-${type}`)
+      input.a-picker(type="radio", name='box', id=`ia-${type}`, checked=(type==='Radio2'))
+      label(for=`ia-${type}`)
         = type
+.test-with-texts-and-icons
+  h3 Texts and icons
+  .m-pickerGroup
+    input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+    label(for=`ib-radio1`)
+      i.a-icon.icon-atom
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze please add -sm class

File: test/chi/components/picker-group.pug [hunk 23]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+      span
+        = `Radio1`
+    input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+    label(for=`ib-radio2`)
+      span
+        = `Radio2`
+      i.a-icon.icon-atom
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze please add -sm class

File: test/chi/components/picker-group.pug [hunk 24]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+    input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+    label(for=`ib-radio3`)
+      i.a-icon.icon-atom
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze please add -sm class

File: test/chi/components/picker-group.pug [hunk 25]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+      span
+        = `Radio3`
+      i.a-icon.icon-atom
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze please add -sm class

File: test/chi/components/picker-group.pug [hunk 26]
```diff
@@ -15,39 +15,80 @@ title: Picker group
...
+
+.test-with-icons
+  h3 Icons
+  .example
+    .m-pickerGroup
+      each type in ['Radio1','Radio2','Radio3']
+        input.a-picker(type="radio", name='radios', id=`ic-${type}`)
+        label(for=`ic-${type}`)
+          i.a-icon.icon-atom
```
Comment (jllr) [inline test/chi/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 27]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
     </div>
 
+h3 Texts and icons
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom"&gt;&lt;/i&gt;</code>.
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup
+      input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+      label(for=`ib-radio1`)
+        i.a-icon.icon-atom
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 28]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+        span
+          = `Radio1`
+      input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+      label(for=`ib-radio2`)
+        span
+          = `Radio2`
+        i.a-icon.icon-atom
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 29]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+      input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+      label(for=`ib-radio3`)
+        i.a-icon.icon-atom
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 30]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+        span
+          = `Radio3`
+        i.a-icon.icon-atom
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 31]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+
+  :code(lang='html')
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio1">
+      <label for="radio1">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 32]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+        <span>Radio1</span>
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio2">
+      <label for="radio2">
+        <span>Radio2</span>
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 33]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio3">
+      <label for="radio3">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 34]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+        <span>Radio3</span>
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 35]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+
+h3 Icons
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom">&lt;/i></code>.
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup
+      each type in ['Radio1','Radio2','Radio3']
+        input.a-picker(type="radio", name='radios', id=`ic-${type}`)
+        label(for=`ic-${type}`)
+          i.a-icon.icon-atom
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 36]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+
+  :code(lang='html')
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio1">
+      <label for="radio1">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 37]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio2">
+      <label for="radio2">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

File: src/website/views/components/picker-group.pug [hunk 38]
```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
...
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio3">
+      <label for="radio3">
+        <i class="a-icon icon-atom"></i>
```
Comment (jllr) [inline src/website/views/components/picker-group.pug]: @janibolkvadze please add -sm class

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/271/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/275/ for more details.

**Pull request merged**